galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -180,3 +186,64 @@ def get(self, request, *args, **kwargs):
         metrics.collection_artifact_download_failures.labels(status=response.status_code).inc()
         raise APIException('Unexpected response from content app. '
                            f'Code: {response.status_code}.')
+
+
+class CollectionVersionMoveViewSet(ViewSet):
```

Comment:
Wondering if this needs to be a separate viewset, of if it could be considered custom "actions" for CollectionVersionViewSet ala https://www.django-rest-framework.org/api-guide/viewsets/#marking-extra-actions-for-routing

I don't have a strong feeling about it either way, so was wondering if custom actions were considered.